harmonic/install_windows_src.md

```diff
@@ -58,6 +67,8 @@ You will still be able to use `TPE` as a physics engine
   libzip qt pybind11 --channel conda-forge
   ```
 
+  This can take tens of minutes.
```

Comment:
Not sure if we want either to suggest mamba or at least to enable conda's mamba based backend (see https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community). I guess conda was originally used here as it works out of the box on Anaconda/miniconda while mamba only works out of the box in miniforge, while for mamba you need to document how to install it. However nowadays conda's mamba backend should work out of the box also in miniconda.

Reply:
Sure, I always wondered why there's conda and mamba =)